### ISSUE TITLE
Basic Sass Example Problem - Running Build From Another Folder

### ISSUE THREAD

**Issue by tteltrab** [lucleray/next-purgecss#7]
Running into an issue trying to get this set up with basic Sass. Not doing anything exciting here:

![image](https://user-images.githubusercontent.com/1487463/47984522-dc4b9780-e0a4-11e8-9a67-a96ea5375c67.png)

Here's my `next.config.js`

![image](https://user-images.githubusercontent.com/1487463/47984555-f1c0c180-e0a4-11e8-91c5-510dfc2b51b6.png)

I think this might be a package problem (would need to investigate the package further to confirm), as my setup is pretty vanilla? Could be on my end, though. Here's my folder structure - pretty basic:

![image](https://user-images.githubusercontent.com/1487463/47984650-3b111100-e0a5-11e8-8f97-b2ac89afc346.png)

Am I missing some requirement/setup piece? Thanks!

**Comment by lucleray** [comment]
I created an example that works here : https://github.com/lucleray/next-purgecss/tree/master/examples/sass

Maybe there's a bug because `main.scss` is in `pages` (i would recommend to move it to a different folder) ?
You also need to install `node-sass` , can you check you have it installed ?

I'm not sure what your bug is 🤔

**Comment by tteltrab** [comment]
Thanks for the example! I think I figured out the problem.

I'm currently running the build one folder up from the `frontend` folder (my `package.json` is running `next build frontend`). Your example works for me, and if I move into the folder and run the build, the package works as well for my example.

![image](https://user-images.githubusercontent.com/1487463/47990103-aa423180-e0b4-11e8-864c-94ade034672a.png)

I think as a result of [using `process.cwd()` in the glob pattern](https://github.com/lucleray/next-purgecss/blob/master/index.js#L18), the glob is looking in the folder running the task, not the folder next is associated with (so `~/my-package/(pages|components)/**/*` instead of `~/my-package/frontend/(pages|components)/**/*`).

My use case would work if the code used [`config.context`](https://webpack.js.org/configuration/entry-context/#context) (getting the path through the webpack config being used by next), instead of `process.cwd()` - I think that might be a solution that works more generally? Happy to make a PR if you agree.

(I pulled down the repo, made the change, did a link, and tested it - fixed my problem, and still works on existing example).
